webapp/src/components/EventStatus.js

```diff
@@ -91,6 +97,19 @@ class EventStatus extends Component {
           
       }
       else if (event.status.application_status === "Withdrawn") {
+        if (event.event_type === "JOURNAL"){
```

Comment:
It seems like we need some bigger changes here to properly cater for multiple submissions with the event statusus. Currently this doesn't make sense that the event application status is "withdrawn" but the status here says "You have submitted your article"